config/initializers/get_into_teaching_api_client.rb

```diff
@@ -0,0 +1,4 @@
+GetIntoTeachingApiClient.configure do |config|
+  config.host = "get-into-teaching-api-dev.london.cloudapps.digital"
```

Comment:
This needs to be an env var

Reply:
Done - Have copied get-into-teaching-app, though we don't obscure the url